script/bootstrap-redhat

```diff
@@ -21,6 +21,8 @@ fi
 
 #yum update -y
 
+yum groupinstall -y "Development Tools"
```

Comment:
I guess it won't make any harm.
It depends more or less what's preinstalled into your system. So for rhel6 7 I admit many devtools preinstalled. But for example vagrant boxes of centos don't have gcc and friends.

So IMO better to have for any yum system.